cai_causal_graph/exceptions.py

```diff
@@ -45,6 +45,9 @@ class NodeInvalidError(Exception):
     class EdgeExistsError(Exception):
         """Error raised when edge already exists in the graph."""
 
+    class ReverseEdgeExistsError(Exception):
+        """Error raised when adding any edge from src to dest where dest to src already exists in the graph."""
```

Comment:
Would shorten to this:

"""Error raised when the reverse edge already exists in the graph."""